pkgs/development/tools/build-managers/alire/default.nix

```
@@ -7,25 +7,27 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "alire";
-  version = "1.2.2";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "alire-project";
     repo = "alire";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-rwNiSXOIIQR1I8wwp1ROVOfEChT6SCa5c6XnTRqekDc=";
+    hash = "sha256-WF7spXwQR04zIGWazUrbCdeLYOzsk8C6G+cfSS6bwdE=";
 
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [ gprbuild gnat ];
 
-  # on HEAD (roughly 2c4e5a3), alire provides a dev/build.sh script. for now,
-  # just use gprbuild.
+  patches = [ ./control-job-count.patch ];
```

fetchpatch is preferred https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#patches

```suggestion
  patches = [
    (fetchpatch {
      name = "control-build-jobs.patch";
      url = "https://github.com/alire-project/alire/commit/2b663e87bdc3d4ba808149bad97bac5c60889b6b.patch";
      hash = "the hash";
    })
  ];
```